server/routerlicious/packages/lambdas/src/nexus/utils.ts

```diff
@@ -65,3 +65,18 @@ export const isWriter = (scopes: string[], mode: ConnectionMode) =>
 	hasWriteAccess(scopes) && mode === "write";
 
 export const getRoomId = (room: IRoom) => `${room.tenantId}/${room.documentId}`;
+export const getClientSpecificRoomId = (clientId: string) => `client#${clientId}`;
+
+export function isSentSignalMessage(obj: any): obj is ISentSignalMessage {
+	return (
+		typeof obj === "object" &&
+		obj !== null &&
+		"content" in obj &&
+		(typeof obj.type === "string" || obj.type === undefined) &&
+		(typeof obj.clientConnectionNumber === "number" ||
+			obj.clientConnectionNumber === undefined) &&
+		(typeof obj.referenceSequenceNumber === "number" ||
```

Comment:
Did you test these variations?

Reply:
Added tests for robustness of these types